apollo-compiler/src/test/graphql/com/example/EpisodeExpected.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public enum Episode {
```

Comment:
As I understood enums are global types, and they are the same across all queries